src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
```

Comment:
it's better to create a package for such classes 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/72727656/199754110-77461d57-29df-4cf2-82c1-05b9d991d0fa.png">